eq-author-api/docker-compose.yml

```diff
@@ -35,6 +35,7 @@ services:
       - JAEGER_SAMPLER_PARAM=1
       - REDIS_DOMAIN_NAME=redis
       - REDIS_PORT=6379
+      - ALLOWED_EMAIL_LIST="@ons.gov.uk, @ons.ext.gov.uk"
```

Comment:
Will this require us to use ONS email addresses for development? Can we set this variable in the production environments instead?

Reply:
Changing this to be an empty string